app/main/views/services.py

```diff
@@ -157,6 +157,17 @@ def update_section(service_id, section):
                 "user",
                 "supplier app")
         except HTTPError as e:
+            errors_map = {}
+            for error in e.response.json()['error'].keys():
+                id = existing_service_content.get_question(error)['id']
+                errors_map[id] = {
+                    'input_name': error,
+                    'question': existing_service_content
+                    .get_question(error)['question'],
```

Comment:
Fixed, but you're right about the line length.  The limit could be a bit more than 80 at least (maybe 120 chars)?